mixer/testdata/config/prometheus.yaml

```diff
@@ -5,98 +5,156 @@ metadata:
-  - name: request_size
+  - name: request_bytes
     instance_name: requestsize.metric.istio-system
     kind: DISTRIBUTION
     label_names:
-    - source_service
+    - source_app
+    - source_name
+    - source_namespace
+    - source_principal
+    - source_workload
+    - source_workload_namespace
     - source_version
-    - destination_service
+    - destination_app
+    - destination_name
+    - destination_namespace
+    - destination_principal
+    - destination_workload
+    - destination_workload_namespace
     - destination_version
+    - destination_service
+    - destination_service_name
+    - destination_service_namespace
+    - request_protocol
     - response_code
+    - connection_mtls
     buckets:
       exponentialBuckets:
         numFiniteBuckets: 8
         scale: 1
         growthFactor: 10
-  - name: response_size
+  - name: response_bytes
     instance_name: responsesize.metric.istio-system
     kind: DISTRIBUTION
     label_names:
-    - source_service
+    - source_app
+    - source_name
+    - source_namespace
+    - source_principal
+    - source_workload
+    - source_workload_namespace
     - source_version
-    - destination_service
+    - destination_app
+    - destination_name
+    - destination_namespace
+    - destination_principal
+    - destination_workload
+    - destination_workload_namespace
     - destination_version
+    - destination_service
+    - destination_service_name
+    - destination_service_namespace
+    - request_protocol
     - response_code
+    - connection_mtls
     buckets:
       exponentialBuckets:
         numFiniteBuckets: 8
         scale: 1
         growthFactor: 10
-  - name: tcp_bytes_sent
+  - name: tcp_sent_bytes_total
     instance_name: tcpbytesent.metric.istio-system
     kind: COUNTER
     label_names:
-    - source_service
+    - source_app
+    - source_name
+    - source_namespace
+    - source_principal
+    - source_workload
+    - source_workload_namespace
     - source_version
-    - destination_service
+    - destination_app
+    - destination_name
+    - destination_namespace
+    - destination_principal
+    - destination_workload
+    - destination_workload_namespace
     - destination_version
-  - name: tcp_bytes_received
+    - destination_service
+    - destination_service_name
+    - destination_service_namespace
+    - connection_mtls
+  - name: tcp_received_bytes_total
     instance_name: tcpbytereceived.metric.istio-system
     kind: COUNTER
     label_names:
-    - source_service
+    - source_app
+    - source_name
+    - source_namespace
+    - source_principal
+    - source_workload
+    - source_workload_namespace
     - source_version
-    - destination_service
+    - destination_app
+    - destination_name
+    - destination_namespace
+    - destination_principal
+    - destination_workload
+    - destination_workload_namespace
     - destination_version
----
-apiVersion: "config.istio.io/v1alpha2"
```

Comment:
Is it intentional to remove these rules? Seems like they are added back anywhere.